jenkins-jobs/pipelines/build-debezium-images.groovy

```diff
@@ -182,6 +181,6 @@ node('Slave') {
             }
         }
     } finally {
-        mail to: ${MAIL_ID}, subject: "${JOB_NAME} run #${BUILD_NUMBER} finished", body: "Run ${BUILD_URL} finished with result: ${currentBuild.currentResult}"
+        mail to: ${MAIL_TO}, subject: "${JOB_NAME} run #${BUILD_NUMBER} finished", body: "Run ${BUILD_URL} finished with result: ${currentBuild.currentResult}"
```

Comment:
`MAIL_TO` is variable so it should be referedd as `MAIL_TO` only. This syntax is only when embedded inside string.

Reply:
Thanks for the clarification. Fixed it.